sdk/messaging/azeventgrid/ci.yml

```diff
@@ -0,0 +1,30 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/messaging/azeventgrid
+    - eng/
+
+pr:
+  branches:
+    include:
+    - main
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/messaging/azeventgrid
+    - eng/
```

Comment:
Did you intent to include this path or is it just copy/pasta?

Reply:
Copy/paste but from the `aztemplate/ci.yml`. Should we remove it there as well?

Were we wanting to auto-rebuild pipelines if we change something about the engineering scripts?

Reply:
IMO it should be removed from the template as we do want to rebuild some pipelines but not all by default (at present we rebuild `azcore`, `internal` and `template`).